inference-engine/src/vpu/graph_transformer/src/model/model.cpp

```diff
@@ -2128,6 +2146,14 @@ void ModelObj::removeUnusedData(const Data& data) {
         _allocator.setNeedToAllocNonIntermData();
     }
 
+    if (const auto dataToShapeEdge = data->_parentDataToShapeEdge) {
```

Comment:
Don't use private member - is there accessor?

Reply:
Done